### ISSUE TITLE
Allow ADAM überscript to take .aws/credentials file

### ISSUE THREAD

**Issue by fnothaft** [BD2KGenomics/toil-scripts#115]
Currently, we just take .boto.

**Comment by hannes-ucsc** [comment]
Yes, `~/.aws/credentials` should be preferred for credentials since it has become the standard and is supported by multiple AWS SDKs (Boto, Java SDK to name two).
